docs2/site/docs/guides/known-issues.md

```diff
@@ -174,67 +176,12 @@ This is done within your database queries; it is not a function of the dataloade
 `CollectionBatchDataLoader` as you would for a one-to-many relationship; then when you are loading
 data from your database within the fetch delegate, use an inner join to retrieve the proper data.
 
-### How to authenticate subscriptions?
```

Comment:
Let's keep H3 with single line of text beneath:

This question has more to do with the transport layer, not GraphQL.NET itself. For more information see [server documentation](https://github.com/graphql-dotnet/server).

Reply:
Why? What about Server project in general? Authentication with ASP.NET Core? JWT authentication?  CORS compatibility? The only reason this question existed is because there was no documentation at all previously. Now there is and it covers all these topics and much more. This particular question is a finer detail of implementation of authentication using the server repo.

Reply:
I did not look into all docs/faq. Do we have anything about subscriptions here? Just if someone go here and does not know about server repo - to guide him in the right direction.

Reply:
I don't see how that is applicable.  It's like having a FAQ titled "how do i change the oil in my car" without having a car.  If we need a general pointer for the server or authentication (e.g. "where can i get a car"), that's fine.  This question is not so.

Reply:
ok